### PR TITLE
Add the version (from config) to the Starting Consul Agent printout

### LINKED ISSUE #1404: Print Consul version on startup
Along similar lines to https://github.com/hashicorp/vault/issues/765, should Consul also print its version when starting up, to aid in troubleshooting?

Currently it prints info that's very similar to what Vault prints:

```
CORP\marca@consul03:/opt/consul$ tail -f /var/log/consul
==> Starting raft data migration...
==> Starting Consul agent...

==> Starting Consul agent RPC...
==> Consul agent running!
         Node name: 'consul03.corp.surveymonkey.com'
        Datacenter: 'monkeytest'
            Server: true (bootstrap: false)
       Client Addr: 0.0.0.0 (HTTP: 8500, HTTPS: -1, DNS: 8600, RPC: 8400)
      Cluster Addr: 10.128.40.149 (LAN: 8301, WAN: 8302)
    Gossip encrypt: false, RPC-TLS: false, TLS-Incoming: false
             Atlas: <disabled>
```

I figure it might as well print the version, just like Vault now does, thanks to @jefferai's fine work on https://github.com/hashicorp/vault/issues/765

### PR DESCRIPTION
**Pull request by richard-hulm**
Fixes #1404

### CODE REVIEW

File: command/agent/command.go [hunk 1]
```diff
@@ -406,7 +406,15 @@ func (c *Command) setupLoggers(config *Config) (*GatedWriter, *logWriter, io.Wri
...
 func (c *Command) setupAgent(config *Config, logOutput io.Writer, logWriter *logWriter) error {
-	c.Ui.Output("Starting Consul agent...")
+	var version string
+
+	version = "v" + config.Version
+
+	if len(config.VersionPrerelease) != 0 {
+		version += " " + config.VersionPrerelease
+	}
+
```
Comment (slackpad) [inline command/agent/command.go]: Can you please also add the revision string so the full version is formatted the same as https://github.com/hashicorp/vault/blob/master/version/version.go#L44-L57.

**Comment by slackpad** [comment]
Hi @richard-hulm thanks for the PR! One small comment and then I think we are good to merge this.

**Comment by richard-hulm** [comment]
@slackpad Updated with that change, cheers.

**Comment by slackpad** [comment]
Thanks!

**Comment by msabramo** [comment]
Thanks, @richard-hulm!

**Pull request merged**